ffi/src/did.rs

```diff
@@ -0,0 +1,59 @@
+use crate::error::{MaybeRustError, Reportable, Try};
+use safer_ffi::prelude::*;
+use signature::suite::ed25519_2020::Mnemonic;
+use tokio::runtime::Runtime;
+use tracing::*;
+
+#[derive_ReprC]
+#[ReprC::opaque]
+#[derive(Clone)]
+pub struct DidDocument {
+    pub(crate) backend: ssi_core::identity::DidDocument,
+}
+
+#[ffi_export]
+pub fn create_identity(
+    rust_error: MaybeRustError,
+    did_method: char_p::Ref<'_>,
+    mnemonic_input: char_p::Ref<'_>,
+) -> Option<repr_c::Box<DidDocument>> {
+    // create_did_doc
+    let _span = error_span!("ffi_ssi").entered();
+    info!(
```

Comment:
@sKudryashov how do we get these logs to carry over to the C side and eventually Dart?

Reply:
My assumption is that it should work out the same way as `println!()` macro, since both are macros. println! works and prints an output, info! most likely will require some pre-configuration on the shim side.